langchain4j-cloud-sql-postgres/src/test/java/dev/langchain/data/document/loader/cloudsql/PostgresLoaderIT.java

```diff
@@ -0,0 +1,179 @@
+package dev.langchain.data.document.loader.cloudsql;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import dev.langchain4j.data.document.Document;
+import dev.langchain4j.data.document.loader.cloudsql.PostgresLoader;
+import dev.langchain4j.engine.PostgresEngine;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PostgresLoaderIT {
+    private static String projectId;
+    private static String region;
+    private static String instance;
+    private static String database;
+    private static String user;
+    private static String password;
+    private static String iamEmail;
+
+    private static PostgresEngine engine;
+    private static Connection connection;
+
+    @BeforeAll
+    public static void beforeAll() throws SQLException {
+        projectId = System.getenv("POSTGRES_PROJECT_ID");
+        region = System.getenv("REGION");
+        instance = System.getenv("POSTGRES_INSTANCE");
+        database = System.getenv("POSTGRES_DB");
+        user = System.getenv("POSTGRES_USER");
+        password = System.getenv("POSTGRES_PASS");
+        iamEmail = System.getenv("POSTGRES_IAM_EMAIL");
+        engine = new PostgresEngine.Builder()
+                .projectId(projectId)
+                .region(region)
+                .instance(instance)
+                .database(database)
+                .user(user)
+                .password(password)
+                .ipType("public")
+                .iamAccountEmail(iamEmail)
+                .build();
+        connection = engine.getConnection();
+    }
+
+    @BeforeEach
+    public void setUp() throws SQLException {
+        createTableAndInsertData();
+    }
+
+    private void createTableAndInsertData() throws SQLException {
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(
+                    "CREATE TABLE test_table (id SERIAL PRIMARY KEY, content TEXT, metadata TEXT, langchain_metadata JSONB)");
```

Comment:
Please add an uuid or identifier to the test_table name to help make the tests more reliable